app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
     permissionChanges: {},
     bibliographicChanges: {},
     valid: Ember.computed.alias('newContributorId'),
+    permissionToggle: false,
+    bibliographicToggle: false,
+    removalToggle: false,
+    stillAdmin: Ember.computed('isAdmin', function() {
+        return this.get('isAdmin');
+    }),
+    addState: 'emptyView', // There are 3 view states on left side of Authors panel. Default state just shows search bar.
```

Comment:
Honestly, the real issue is that we're presenting several pages inside a single route, which is a bit different from ember's core assumptions about how route is tied to the state of the application.

I'd need to experiment a bit more with the routing mechanisms to see what we could do, and just haven't had the bandwidth to explore these sorts of questions. I still think the notion of nested routing (a `/search/` child route) has potential, but there are questions about what happens when you change things in the child route that we don't know enough about yet. 

Ember workarounds that show three "pages" in one route are very common- the nested router isn't a solution to everything. But I think there's potential we haven't tapped yet.

Somewhere in the back of my mind is the tabbed diagram here:
http://fromrailstoember.com/9-nested-routes-equals-nested-ui/